components/encumbered/lms/Makefile

```diff
@@ -0,0 +1,119 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
+# Copyright 2020 Rouven Weiler
+#
+
+BUILD_BITS=			32
+BUILD_STYLE=			archive
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME =		lms
+COMPONENT_FMRI =		service/multimedia/
+COMPONENT_VERSION =		7.9.2
+COMPONENT_PROJECT_URL =		https://github.org/Logitech/slimserver.git
+COMPONENT_SUMMARY =		Logitech Media Server
+COMPONENT_LICENSE =		GPLv2
+COMPONENT_LICENSE_FILE =	$(COMPONENT_NAME).license
+COMPONENT_CLASSIFICATION =	System/File System
+
+COMPONENT_REPO_NAME =		slimserver
+COMPONENT_GIT_COMMIT_HASH =	9b9f2d148bae0021913bbe9642cce8dc83a3ad00
+COMPONENT_SRC =			$(COMPONENT_REPO_NAME)-$(COMPONENT_GIT_COMMIT_HASH)
+COMPONENT_ARCHIVE =		$(COMPONENT_REPO_NAME)-$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH =	sha256:0e787e633ce002f5fe59da4ba71cfbfe3b43ece5a40490b58c88c2b44061fbc5
+COMPONENT_ARCHIVE_URL =		https://github.com/Logitech/$(COMPONENT_REPO_NAME)/archive/$(COMPONENT_GIT_COMMIT_HASH).tar.gz
+
+COMPONENT_REPO_NAME_1 =		slimserver-vendor
+COMPONENT_GIT_COMMIT_HASH_1 =	6265ea5b0ab9952af187602e848bbe318f4e3cc6
+COMPONENT_SRC_1 =		$(COMPONENT_REPO_NAME_1)-$(COMPONENT_GIT_COMMIT_HASH_1)
+COMPONENT_ARCHIVE_1 =		$(COMPONENT_REPO_NAME_1)-$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH_1 =	sha256:055d9e31c79e683da53f09e9fb02833b29ed12d4168ddc59d46447d277992da8
+COMPONENT_ARCHIVE_URL_1 =	https://github.com/Logitech/$(COMPONENT_REPO_NAME_1)/archive/$(COMPONENT_GIT_COMMIT_HASH_1).tar.gz
+
+PERL_VERSION =			5.22
+
+BUILD_TARGET =			$(BUILD_DIR)/$(COMPONENT_NAME)/.built $(BUILD_DIR)/$(COMPONENT_NAME)-vendor/.built
+
+INSTALL_TARGET =		$(BUILD_DIR)/$(COMPONENT_NAME)/.installed
+
+include $(WS_MAKE_RULES)/common.mk
+
+BUILDME_ENV +=                  MAKE=$(GMAKE)
+BUILDME_ENV +=			PATH=$(dir $(CC)):$(SPRO_VROOT)/bin:$(PATH)
```

Comment:
Why do we need $(SPRO_ROOT)/bin here?

Reply:
Well, that was the copy from makemaker.mk to make it run. I would be glad to delete it... and hoped so. I thought it was necessary for perl to work well...

Reply:
Removed $(SPRO_VROOT).